api.go

```diff
@@ -373,15 +373,30 @@ func (api *API) delSilence(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *API) listSilences(w http.ResponseWriter, r *http.Request) {
-	sils, err := api.silences.All()
+	var (
+		num    = r.FormValue("n")
```

Comment:
limit + offset would be fine as well. I like the idea of using page (starting at 1) externally and offset (starting at 0) internally.

Do we actually need a configurable limit to begin with? Could we just go with 25 for now and keep it simple until we have enough user requests to make it configurable?